README.md

```diff
@@ -79,13 +79,33 @@ Update required configuration variables located in `readml/config/config_local.c
 
 One way of using the package is to run the `interpret` function which takes care of explaining model behaviour .
 
-You need to update required configuration variables located in `readml/config/config_local.cfg` before.
+You may update required configuration variables located in `readml/config/config_local.cfg` before.
 
 For instance , using **partial dependency plots** for global interpretability on ML model:
 ```python
 from readml.main import interpret
 interpret(interpret_type="global", use_pdp_ice=True, use_ale=False, use_shap=False)
 ```
+
+In the other way, you can give as an input a dictionnary into `interpret` function with the same config keys located in `readml/config/`, as an example, after the run of pytest :
+
```

Comment:
Proposition de reformulation :

You can also use the `interpret` function without filling in the configuration file. To do so, you need to pass a config dictionary, as additional parameter to the function, with the same config keys located in `readml/config/` (you will find templates for each usecase). 
Here is an example using the outputs of unit tests :